### PR TITLE
fix: files

### LINKED ISSUE #509: [MAINTENANCE] Missing or outdated AUTHORS file in ansys/pyansys-dev-guide
Please add the AUTHORS file to the repository. The file might either be missing or is an outdated version of the expected file.

See [here](https://github.com/ansys-internal/pyansys-maintenance-automation/blob/main/scripts/file_references/AUTHORS) an up-to-date reference to a properly formatted file.

---

This issue was created by a bot. If you have any questions, please ping the PyAnsys Core team for assistance. To ping the team write a comment in this issue with the following mention: `@ansys/pyansys-core`.

If you want this repository to be excluded from this automated maintenance process, please let us know by [filling in the opt-out request form](https://github.com/ansys-internal/pyansys-maintenance-automation/issues/new?labels=&projects=&template=opt-out.yml&title=Request+to+opt-out+from+automated+maintenance+for+ansys%2Fpyansys-dev-guide).

### LINKED ISSUE #510: [MAINTENANCE] Missing or outdated CONTRIBUTORS.md file in ansys/pyansys-dev-guide
Please add the CONTRIBUTORS.md file to the repository. The file might either be missing or is an outdated version of the expected file.

See [here](https://github.com/ansys-internal/pyansys-maintenance-automation/blob/main/scripts/file_references/CONTRIBUTORS.md) an up-to-date reference to a properly formatted file.

---

This issue was created by a bot. If you have any questions, please ping the PyAnsys Core team for assistance. To ping the team write a comment in this issue with the following mention: `@ansys/pyansys-core`.

If you want this repository to be excluded from this automated maintenance process, please let us know by [filling in the opt-out request form](https://github.com/ansys-internal/pyansys-maintenance-automation/issues/new?labels=&projects=&template=opt-out.yml&title=Request+to+opt-out+from+automated+maintenance+for+ansys%2Fpyansys-dev-guide).

### PR DESCRIPTION
**Pull request by germa89**
As the title.

Close #510
Close #509

### CODE REVIEW

**Review by germa89 — COMMENTED** [review]
I do not know who is the project lead for this library. Also, feel free to add anyone else to the contributors list.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Individual Contributors
+
+* [Alex Kaszynski](https://github.com/akaszynski)
+* [Camille Latapie](https://github.com/clatapie)
+* [Jorge Martínez](https://github.com/jorgepiloto)
+* [Kathy Pipet](https://github.com/PipKat)
+* [Maxime Rey](https://github.com/MaxJPRey)
+* [Revathy Venugopal](https://github.com/Revathyvenugopal162)
+* [German Martinez Ayuso](https://github.com/germa89)
```
Comment (RobPasMue) [inline CONTRIBUTORS.md:15]:
```suggestion
# Contributors

## Project Lead

* [Camille Latapie](https://github.com/clatapie)
* [German Martinez Ayuso](https://github.com/germa89)
* [Jorge Martínez](https://github.com/jorgepiloto)
* [Maxime Rey](https://github.com/MaxJPRey)
* [Revathy Venugopal](https://github.com/Revathyvenugopal162)
* [Roberto Pastor](https://github.com/RobPasMue)

## Individual Contributors

* [Alex Kaszynski](https://github.com/akaszynski)
* [Kathy Pippert](https://github.com/PipKat)

```

I would add all of the @ansys/pyansys-core team members that have contributed

Comment (germa89) [reply CONTRIBUTORS.md:15]: As long as the bot is happy about it...

Comment (RobPasMue) [reply CONTRIBUTORS.md]: Yep - we accept several project leads in the regex :)

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**